oauth2/oauth2-client/src/test/java/org/springframework/security/oauth2/client/registration/TestClientRegistrations.java

```diff
@@ -55,6 +55,21 @@
 				.clientSecret("client-secret");
 	}
 
+	public static ClientRegistration.Builder clientRegistration3() {
```

Comment:
Move this to be a private static method in `DefaultOAuth2AuthorizationRequestResolverTests`

Reply:
OK